cmd/secret-vault/flag.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+
+	"github.com/go-vela/secret-vault/vault"
+)
+
+// helper function to load the flags into
+// the plugin configuration
+func flags() []cli.Flag {
+	f := []cli.Flag{
+		&cli.StringFlag{
+			EnvVars: []string{"PARAMETER_LOG_LEVEL", "VELA_LOG_LEVEL", "ARTIFACTORY_LOG_LEVEL"},
```

Comment:
Could we either remove `ARTIFACTORY_LOG_LEVEL` or replace it with a proper name?